### DISCUSSION TITLE
MFD screens reset to default when select key is double clicked

### DISCUSSION THREAD

**Discussion by KBNOtaku** [magknight/787aviatorsTracker#652] [Q&A] [answered]
**Describe the bug**
When double clicking the CHKL button MFD jumps back to ND right side?

**To Reproduce**
Set the MFD in any config where ND is left sided and double click the CHKL button.

**Expected behaviour**
double click the CHKL button should not reset the MFD?

**Video**
https://drive.google.com/file/d/1VhJL5VjJIKWYRqTMeH_kqKfACqyJxL2m/view?usp=sharing

**Desktop (please complete the following information):**
 - OS:  Windows 10 ]
 - X-Plane version [ 11.55r2 ]
 - Version [ 1.7.18-alpha6 ]

**Comment by aeroniemi** [comment, marked as answer]
That's performing as expected! In reality, you click once to select the display, and again to go back to the "default" state for that screen
the only thing that is perhaps wrong there is whether the other half should go to SELECT or to it's default state (if available) - the latter would probably end up being too complicated as it'd require iteratively checking display states, which isn't built in at present (and seems like rather a lot of effort for not much gain!)
